src/button.js

```diff
@@ -58,7 +58,8 @@ function distributeClassPrefix(prefix, classAttr) {
 
 // DOM Event Listeners
 Button.prototype.events = View.prototype.events.extended({
-    click: '_execute'
+    click: '_execute',
+    keyup: '_execute'
```

Comment:
It seems to me that _execute should have a check of which keyup happened. Tabbing to a button and pressing "t" shouldn't do anything, but I think it does, as _execute currently stands.